pan3d/dataset_builder.py

```diff
@@ -327,6 +362,16 @@ def slicing(self) -> Dict[str, List]:
         integers or floats representing start value, stop value, and step.
         """
         return self._algorithm.slicing
+    
+    @property
+    def extents(self) -> map:
+        extents = {}
```

Comment:
Does not look like a vtk extents?

Reply:
I made so intentionally as our dimensions are named and the users might be interested in getting extents by the keys for the respective dimension. Also in case of, also this allows us to only return results for as many dimensions as the data has instead of always returning 6 doubles. 
But I can make it `VTK` style so that VTK users find it familiar. Thoughts?

Reply:
That is ok, I was just making sure it was intentional. 

You should add a comment on the python property to explain what you get.